avsomr/avsomrlocal.cpp

```diff
@@ -0,0 +1,289 @@
+#include "avsomrlocal.h"
+
+#include <QFileInfo>
+#include <QEventLoop>
+#include <QProcess>
+
+#include "mscore/preferences.h"
+
+#include "avslog.h"
+#include "avsomrlocalinstaller.h"
+
+namespace {
+
+static const QString AVS_EXT_NAME("avsomr_local");
+
+#ifdef Q_OS_WIN
+
+static const QString SHELL("cmd");
+static const QStringList SHELL_ARGS("/c");
+static const QString AVS_MAKE_SCRIPT("make_avs_win.bat");
+
+#elif defined(Q_OS_MAC)
+
+static const QString SHELL("sh");
+static const QStringList SHELL_ARGS;
+static const QString AVS_MAKE_SCRIPT("make_avs_mac.sh");
+
+#else
+
+static const QString SHELL("sh");
+static const QStringList SHELL_ARGS;
+static const QString AVS_MAKE_SCRIPT("make_avs_lin.sh");
+
+#endif
+}
+
+using namespace Ms::Avs;
+
+AvsOmrLocal::AvsOmrLocal()
+      {
+
+      }
+
+AvsOmrLocal::~AvsOmrLocal()
```

Comment:
better in the .h?

Reply:
https://github.com/musescore/MuseScore/pull/5692#discussion_r386407502

Reply:
hence my ? ;-)